faf.spec.in

```diff
@@ -188,7 +186,7 @@ A plugin for %{name} implementing archive-reports action
 %package action-create-problems
 Summary: %{name}'s create-problems plugin
 Requires: %{name} = %{faf_version}
-Requires: satyr-python >= 0.9
+Requires: %{satyr_dep}
```

Comment:
No. This was added in 0c3c394eb802366b4d47989c04cd36a8e4d4abf6 and this subpackage really require only satyr-python >= 0.9 It does not matter that somewhere else we require satyr-python > 0.13-1 or satyr-python >= 0.16.

Reply:
0.9 comes from 2013. So I think that the version can be safely removed. I.e.: `Requires: satyr-python`

Reply:
I made the change to be consistent with https://github.com/abrt/faf/commit/b62cd1c23453cfa77215faa00e7ecff8e57f6368